projects/plugins/jetpack/modules/videopress/class.videopress-edit-attachment.php

```diff
@@ -210,6 +212,14 @@ public function fields_to_edit( $fields, $post ) {
 			'html'  => $this->display_rating( $info ),
 		);
 
+		if ( 1 !== 1 /* Temporarily disabled. */ ) {
```

Comment:
Just confirming we still want this disabled?

Reply:
We don't good catch!!